lms/static/js/jquery.gradebook.js

```diff
@@ -1,24 +1,19 @@
-
-
-
 var Gradebook = function($element) {
-	var _this = this;
-	var $element = $element;
+  "use strict";
 	var $body = $('body');
 	var $grades = $element.find('.grades');
 	var $studentTable = $element.find('.student-table');
 	var $gradeTable = $element.find('.grade-table');
 	var $search = $element.find('.student-search-field');
 	var $leftShadow = $('<div class="left-shadow"></div>');
-	var $rightShadow = $('<div class="right-shadow"></div>');
```

Comment:
any particular reason for removing this? (I'm not a fan of the shadows myself, but I didn't want to change too much)